app/_hub/kong-inc/ai-prompt-decorator/_metadata.yml

```diff
@@ -0,0 +1,22 @@
+name: AI Prompt Decorator
+search_aliases:
+  - ai
+  - llm
+  - artificial
+  - intelligence
+  - language
+  - model
+dbless_compatible: yes
+free: true
+enterprise: true
+paid: true
+konnect: true
+network_config_opts: All
+notes: --
+categories:
+  - ai
+publisher: Kong Inc.
+desc: Add an array of llm/v1/chat messages to either the start or end of a caller's chat history
```

Comment:
```suggestion
desc: Prepend or append an array of llm/v1/chat messages to a user's chat history
```

Reply:
`llm/v1/chat` is a type name, we should code format them across the board.

Reply:
Unfortunately, the layout breaks if I try that in the short descriptions: 
![Screenshot 2024-02-14 at 9 10 34 AM](https://github.com/Kong/docs.konghq.com/assets/54370747/1b1800cb-683b-480b-8bef-8cec630ed677)

Reply:
understood, unfortunate